Game.java

```diff
@@ -136,16 +255,40 @@ public String getInfo(Game newGame){
 	}*/
 }
 
-class bigPlayer{
+class Map{
+	String mapName = "";
+
+	Map(){
+	}
+
+	Map(String map){
+		this.mapName = map;
+	}
+
+	public String getMapName(){
+		return this.mapName;
+	}
+}
+
+class Player{
 	String name = "";
 	boolean isCaptain = false;
+	int playerId = 0;
+
+	Player(){
+	}
```

Comment:
why do you keep making empty constructors?

Reply:
This one I actually need because there are times when I need to make an empty player object (ie. when I dont know what is going to be stored in the player object yet)

Reply:
why would there be an empty player? doesnt a player object get created when a player connects? at which point you know everything you need to know about them